udata/commands/fixtures.py

```diff
@@ -13,23 +14,85 @@
 log = logging.getLogger(__name__)
 
 
+DATASET_SLUGS = []
+
+
+DATASET_URL = '/api/1/datasets'
+ORG_URL = '/api/1/organizations'
+REUSE_URL = '/api/1/reuses'
+COMMUNITY_RES_URL = '/api/1/datasets/community_resources'
+
+
+DEFAULT_FIXTURE_FILE = 'https://github.com/opendatateam/udata-fixtures/blob/main/results.json'  # noqa
+
+
 @cli.command()
-@click.option('-d', '--datasets', type=int, default=5,
-              help='Number of datasets to generate')
-@click.option('-r', '--reuses', type=int, default=1,
-              help='Number of reuses by dataset')
-def generate_fixtures(datasets, reuses):
-    '''Build sample fixture data (users, datasets and reuses).'''
-    user = UserFactory()
-    log.info('Generated user "{user.email}".'.format(user=user))
-
-    organization = OrganizationFactory(members=[Member(user=user)])
-    log.info('Generated organization "{org.name}".'.format(org=organization))
-
-    for _ in range(datasets):
-        dataset = VisibleDatasetFactory(organization=organization)
-        DiscussionFactory(subject=dataset, user=user)
-        ReuseFactory.create_batch(reuses, datasets=[dataset], owner=user)
-
-    msg = 'Generated {datasets} dataset(s) with {reuses} reuse(s) each.'
-    log.info(msg.format(**locals()))
+@click.argument('data-source')
+def generate_fixtures_file(data_source):
+    '''Build sample fixture file based on datasets slugs list (users, datasets, reuses).'''
+    json_result = []
+
+    with click.progressbar(DATASET_SLUGS) as bar:
+        for slug in bar:
+            json_fixture = {}
+
+            json_dataset = requests.get(f'{data_source}{DATASET_URL}/{slug}/').json()
+            del json_dataset['uri']
+            del json_dataset['page']
+            del json_dataset['last_update']
+            del json_dataset['license']
+            json_org = json_dataset.pop('organization')
+            json_resources = json_dataset.pop('resources')
+            for res in json_resources:
+                del res['latest']
+                del res['preview_url']
+                del res['last_modified']
+            json_fixture['resources'] = json_resources
+            json_fixture['dataset'] = json_dataset
+
+            json_org = requests.get(f"{data_source}{ORG_URL}/{json_org['id']}/").json()
```

Comment:
If we have two datasets from same organization, we would try to generate the organization twice later though?